code/go/0chain.net/smartcontract/storagesc/shut_down.go

```diff
@@ -0,0 +1,83 @@
+package storagesc
+
+import (
+	cstate "0chain.net/chaincore/chain/state"
+	"0chain.net/chaincore/transaction"
+	"0chain.net/core/common"
+)
+
+func (ssc *StorageSmartContract) shutDownBlobber(
```

Comment:
how is this status used

Reply:
It is used to decide whether to use the provider for any new activities. In the case of a blobber it means no new allocations.
https://github.com/0chain/0chain/blob/provider_changes/code/go/0chain.net/smartcontract/storagesc/models.go#L867